src/unix/atomic-ops.h

```diff
@@ -53,6 +53,8 @@ UV_UNUSED(static int cmpxchgi(int* ptr, int oldval, int newval)) {
 UV_UNUSED(static void cpu_relax(void)) {
 #if defined(__i386__) || defined(__x86_64__)
   __asm__ __volatile__ ("rep; nop");  /* a.k.a. PAUSE */
+#elif (defined(__arm__) && __ARM_ARCH >= 7) || defined(__aarch64__)
```

Comment:
```suggestion
#elif defined(__ARM_ARCH_7__) || defined(__aarch64__)
```
If Cortex-A supports `yield` but e.g. Cortex-R doesn't, then `__ARM_ARCH_7A__` would be more appropriate. Some quick googling suggests all ARMv7 cores support it though.

Reply:
on my rasp this is not defined. ie __ARM_ARCH_8A__

Reply:
I suppose that makes sense. I guess it doesn't matter much which one is used so LGTM.